### PR TITLE
Updated SDK init api

### PR DESCRIPTION
**Pull request by ashinagawa**
This commit will update to use new Vungle SDK init API.

IOS-2890

### CODE REVIEW

File: adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m [hunk 1]
```diff
@@ -96,8 +96,22 @@ - (void)initWithAppId:(nonnull NSString *)appId
...
+    GADMAdapterVungleAdType adType = [delegate adapterAdType];
+    if (adType == GADMAdapterVungleAdTypeBanner || adType == GADMAdapterVungleAdTypeShortBanner || adType == GADMAdapterVungleAdTypeLeaderboardBanner) {
+      priorityPlacementAdSize = [self getVungleBannerAdSizeType:adType];
+    }
+    [initOptions setObject:[NSNumber numberWithInteger:priorityPlacementAdSize] forKey:VungleSDKInitOptionKeyPriorityPlacementAdSize];
+  }
+     
   NSError *err = nil;
-  [sdk startWithAppId:appId error:&err];
+    [sdk startWithAppId:appId options:initOptions error:&err];
```
Comment (YueVungle) [inline adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m:114]: @ashinagawa An extra space, thanks ^_^

File: adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m [hunk 2]
```diff
@@ -96,8 +96,22 @@ - (void)initWithAppId:(nonnull NSString *)appId
   //Disable refresh functionality for all banners
   [[VungleSDK sharedSDK] disableBannerRefresh];
 
+  // Set init options for priority placement
+  NSMutableDictionary *initOptions = [NSMutableDictionary dictionary];
+  if (delegate) {
+    NSString *priorityPlacementID = delegate.desiredPlacement ? delegate.desiredPlacement : @"";
```
Comment (YueVungle) [inline adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m:102]: @ashinagawa Hi Aki, I have checked the `desiredPlacement.length` in `GADMAdapterVungleInterstitial` and `GADMAdapterVungleRewardedAd` classes, I believe we don't need to check `delegate.desiredPlacement is nil ` here. Thanks a lot. ^_^

File: adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m (hunk 1 above)
Comment (ashinagawa) [reply adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m]: Updated.

File: adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m (hunk 2 above)
Comment (ashinagawa) [reply adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m]: Updated ;-)

**Pull request merged**